INN/umbrella-mstoday
# Adjust width of inserted ads inside post content

## Changes

This pull request makes the following changes:

- Adjusts the width of the SCAIP widget areas inside post content so they are the same width as the post content

![Screen Shot 2020-03-03 at 12 07 25 PM](https://user-images.githubusercontent.com/18353636/75800628-e092ce00-5d47-11ea-8718-fdc698c9d55a.png)
![Screen Shot 2020-03-03 at 12 07 18 PM](https://user-images.githubusercontent.com/18353636/75800630-e12b6480-5d47-11ea-9fac-aa5b087dc5fc.png)

## Why



For #68

> As for the widget area (called Inserted Ad 2) where we'd like to place the newsletter signup in story posts (likely using the Forms widget), we're having the same issue with the width as we're having in the Inserted Ad 1 widget area where we'd like to include the Read Next widget. I'm thinking, by adjusting the width of those widget areas so they fit inside the story post container, it should resolve the issue (sort of two birds, one stone, I think).

## Testing/Questions

Features that this PR affects:

- SCAIP widget areas inside post content


Questions that need to be answered before merging:


Steps to test this PR:

1. Add some widgets to each `Inserted Ad Position 1` and `Inserted Ad Position 2` widget areas
2. View on desktop and mobile and make sure the widget area doesn't extend further than the post content
